deploy/charts/harvester/values.yaml

```diff
@@ -0,0 +1,253 @@
+  ## defaults to "true".
+  ##
+  enabled: true
+
+## Specify the parameters to override the sub-chart.
+##
+kubevirt:
+
+  ## Specify to install KubeVirt CRD instance,
+  ## defaults to "true".
+  ##
+  enabled: true
+
+## Specify the parameters to override the sub-chart.
+##
+cdi-operator:
+
+  ## Specify to install CDI Operator,
+  ## defaults to "true".
+  ##
+  enabled: true
+
+## Specify the parameters to override the sub-chart.
+##
+cdi:
+
+  ## Specify to install CDI CRD instance,
+  ## defaults to "true".
+  ##
+  enabled: true
+
+## Specify the parameters to override the sub-chart.
+##
+minio:
+
+  ## Specify to install Minio,
+  ## defaults to "true".
+  ##
+  enabled: true
+
+  ## Specify the full name of Minio.
+  ##
+  fullnameOverride: "minio"
+
+  ## Specify the resources of Minio container.
+  ##
+  resources:
+    requests:
+      memory: 256m
+
+###########################################################################
+###########################################################################
+##                    Default values for Harvester                       ##
+###########################################################################
+###########################################################################
+
+## Provide a name in place of labels.
+##
+nameOverride: ""
+
+## Provide a name to substitute for the full names of resources.
+##
+fullnameOverride: ""
+
+## Specify the replicas of workload.
+##
+replicas: 1
+
+## Specify the security context of workload.
+##
+securityContext: {}
+
+## Specify the node selector of workload.
+##
+nodeSelector: {}
+
+## Specify the tolerations of workload.
+##
+tolerations: []
+#  - key: CriticalAddonsOnly
+#    operator: Exists
+
+## Specify the update strategy of workload.
+##
+strategy:
+  type: RollingUpdate
+
+## Specify the parameter of containers.
+##
+containers:
+
+  ## Specify the parameter of harvester-apiserver container.
+  ##
+  apiserver:
+
+    ## Specify the image.
+    ##
+    image:
+
+      ## Specify the pull policy of image.
+      ##
+      imagePullPolicy: Always
+
+      ## Specify the repository of image.
+      ##
+      repository: rancher/vm
+
+      ## Specify the tag of image.
+      ##
+      tag: master-head
+
+    ## Specify the command.
+    ##
+    command: []
+
+    ## Specify the args.
+    ##
+    args: []
+
+    ## Specify the env.
+    ##
+    env: []
+    #  - name: OPERATOR_IMAGE
+    #    value: xxx
+
+    ## Specify the liveness probe.
+    ##
+    livenessProbe: {}
+
+    ## Specify the readiness probe.
+    ##
+    readinessProbe: {}
+
+    ## Specify the resources.
+    ##
+    resources:
+      limits:
+        cpu: 500m
+        memory: 500Mi
+      requests:
+        cpu: 250m
+        memory: 256Mi
+
+    ## Specify the gateway to access Minio-like VM image storage service which is compatible with S3,
+    ## only valid if Minio sub-charts is disabled in the same chart release.
+    ##
+    vmImageStorageGateway:
```

Comment:
Is it a common word using `gateway` in this use case?

Reply:
Inspired by Minio values.